knowledge_graphs/knowledge_graph_validator.py

```
@@ -1,41 +1,42 @@
+import os
+import sys
 from dataclasses import dataclass, field
-from enum import Enum
 from typing import Any
 
-from ai_script_analyzer import AnalysisResult, AttributeAccess, ClassInstantiation, FunctionCall, ImportInfo, MethodCall
-from neo4j import AsyncGraphDatabase
+# Add parent directory to path for imports
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from knowledge_graphs.ai_script_analyzer import (
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)